package.json

```diff
@@ -31,7 +31,7 @@
     "platformicons": "0.1.0",
     "po-catalog-loader": "^1.2.0",
     "query-string": "2.4.2",
-    "raven-js": "3.5.0",
+    "raven-js": "git+https://github.com/getsentry/raven-js.git#22bebbdbb619183978761dbb68fcb59ae8b3b099",
```

Comment:
I personally prefer the `getsentry/raven-js#22bebbdbb619183978761dbb68fcb59ae8b3b099` syntax myself.